kentikapi/credentials.go

```diff
@@ -0,0 +1,78 @@
+package kentikapi
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+// ReadCredentialsFromEnv reads and returns (email, token) pair from environment variables.
+func ReadCredentialsFromEnv() (authEmail, authToken string) {
+	authEmail, _ = os.LookupEnv("KTAPI_AUTH_EMAIL")
+	authToken, _ = os.LookupEnv("KTAPI_AUTH_TOKEN")
+	return authEmail, authToken
+}
+
+// ReadCredentialsFromProfile reads and returns (email, token) pair from profile config file.
+func ReadCredentialsFromProfile() (authEmail, authToken string, err error) {
+	var result map[string]string
+	profile := "default"
+	filename := getFilename(profile)
+	jsonFile, err := os.Open(filepath.Clean(filename))
+
+	if err != nil {
+		return "", "", err
+	}
+	defer func() {
+		cerr := jsonFile.Close()
+		if err == nil {
+			err = cerr
+		}
+	}()
+	byteValue, err := ioutil.ReadAll(jsonFile)
+	if err != nil {
+		return "", "", err
+	}
+
+	err = json.Unmarshal(byteValue, &result)
+	if err != nil {
+		return "", "", err
+	}
+	for _, s := range []string{"email", "api-key"} {
+		if _, ok := result[s]; !ok {
+			return "", "", fmt.Errorf("no '%s' in config file: %s", s, filename)
+		}
+	}
+	return result["email"], result["api-key"], nil
+}
+
+func getFilename(profile string) string {
+	home, ok := os.LookupEnv("KTAPI_HOME")
+	if !ok || home == "" {
+		home, ok = os.LookupEnv("HOME")
+		if !ok || home == "" {
+			home = "."
+		}
+	}
+	filename, ok := os.LookupEnv("KTAPI_CFG_FILE")
+	if !ok || filename == "" {
+		return filepath.Join(home, ".kentik", profile)
+	}
+	return filename
+}
+
+// GetCredentials reads and returns (email, token) pair from environment variables or profile config file.
```

Comment:
```suggestion
// GetCredentials reads and returns (email, token) pair from environment variables or profile config file.
// Profile config file path is ${HOME}/.kentik/default by default. It can be overridden by KTAPI_CFG_FILE env variable // or combination of KTAPI_HOME and KTAPI_PROFILE variables forming path "${KTAPI_HOME}/.kentik/${KTAPI_PROFILE}".
```

Reply:
Still to do

Reply:
@jdziurzynski Still to do. Order of credentials sources given by Mateusz and profile file format will be enough

Reply:
@jdziurzynski Please add profile file format (e.g. example JSON) to the docstring.

Reply:
Done